src/Reflection/ReflectionType.php

```diff
@@ -3,11 +3,22 @@
 
 namespace Roave\BetterReflection\Reflection;
 
-use phpDocumentor\Reflection\Type;
-use phpDocumentor\Reflection\Types;
-
 class ReflectionType
 {
+    private const BUILT_IN_TYPES = [
+        'int'      => true,
```

Comment:
Use `null` as value, since boolean involves two possible values, and all we want is existence/non-existence